grammar/fr/src/rules.rs

```diff
@@ -314,12 +314,6 @@ pub fn rules_duration(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
              time_check!(),
              |duration, _, time| duration.value().after(time.value())
     );
-    b.rule_3("<duration> apres <time>",
```

Comment:
Why did you remove this rule but let the rule <duration> avant <time>?

Reply:
That was just a doublon, the rule was there twice